src/languages/en.ts

```diff
@@ -3217,6 +3217,25 @@ export default {
     subscription: {
         mobileReducedFunctionalityMessage: 'You can’t make changes to your subscription in the mobile app.',
         billingBanner: {
+            outdatedInfo: 'Your payment info is outdated',
+            updatePaymentInformation: 'Please update your payment information.',
+            updateCardDataByDate: ({date}) => `Update your payment card by ${date} to continue using all of your favorite features.`,
+            paymentPastDuePayByDate: ({date}) => `Your payment is past due. Please pay your invoice by ${date} to avoid service interruption`,
```

Comment:
```suggestion
            paymentPastDuePayByDate: ({date}) => `Your payment is past due. Please pay your invoice by ${date} to avoid service interruption.`,
```